### ISSUE TITLE
JLabel html rendering changes font size

### ISSUE THREAD

**Issue by cmadsen** [JFormDesigner/FlatLaf#912]
JLabel html rendering changes font size with java 17 using flatlaf 3.5.2

```java
JLabel l1 = new JLabel("Destination");
String htmlDestination = "<HTML><A HREF=http://destination>Destination</A></HTML>";
JLabel l2 = new JLabel(htmlDestination);
```
![unnamed](https://github.com/user-attachments/assets/fe665288-b62a-4c5e-8c14-1c573dfb255b)

Without flatlaf

![unnamed](https://github.com/user-attachments/assets/a184a0be-98c3-482f-9a7e-a411ee8df4c0)

The following code can be used to reproduce the issue:

```java
import static javax.swing.SwingUtilities.*;

import java.awt.Font;

import javax.swing.JLabel;
import javax.swing.JPanel;
import javax.swing.UIManager;

import org.jdesktop.swingx.JXFrame;

import com.formdev.flatlaf.FlatLightLaf;

import net.miginfocom.swing.MigLayout;

public class FlatLafTextFieldHtmlIssue extends JXFrame {

	public FlatLafTextFieldHtmlIssue() {
		try {
			UIManager.setLookAndFeel(new FlatLightLaf());
		} catch (Exception e) {
			e.printStackTrace();
		}
		JPanel p = new JPanel(new MigLayout());
		JLabel l1 = new JLabel("Destination");
		p.add(l1, "wrap");
		String htmlDestination = "<HTML><A HREF=http://destination>Destination</A></HTML>";
		JLabel l2 = new JLabel(htmlDestination);
		p.add(l2, "wrap");
		getContentPane().add(p);
		Font f1 = l1.getFont();
		Font f2 = l2.getFont();
		System.out.println(f1);
		System.out.println(f2);
		setTitle("Flat Laf Html Text Issue");
		setDefaultCloseOperation(EXIT_ON_CLOSE);
		pack();
		setVisible(true);
	}

	public static void main(String[] args) {
		invokeLater(() -> {
			new FlatLafTextFieldHtmlIssue();
		});
	}

}
```

**Comment by cmadsen** [comment]
Fedora 40, KDE, Xorg session

**Comment by DevCharly** [comment]
Fixed in latest `3.5.3-SNAPSHOT`: https://github.com/JFormDesigner/FlatLaf#snapshots
